docs/source/markdown/options/update-connection.md

```diff
@@ -0,0 +1,12 @@
+####> This option file is used in:
+####>   podman machine init, machine start
+####> If file is edited, make sure the changes
+####> are applicable to all of those.
+#### **--update-connection**, **-u**
+
+When used in conjunction with `podman machine init --now` or `podman machine start`, this option sets the
+associated machine system connection as the default.
+
+If the value is set to true, the machine connection will be set as the system default.
+If the value is set to false, the system default will be unchanged.
```

Comment:
It is worth pointing out that this must be `--update-connection=false` not `--update-connection false` which may not be obvious to users if you talk about value.

Reply:
right, added.